src/Simulation.ts

```diff
@@ -0,0 +1,59 @@
+import { DCRGraph } from "./dcr";
+import { parser } from "./dcr_parser"
+
+class Simulation {
+    graph: DCRGraph;
+    isRunning: boolean = false;
+    id: number;
+    log: any;
+    users: any[];
+    startTime: Date;
+    stopTime: Date;
+
+    constructor(input: string) {
```

Comment:
I see you don't instantiate a Log with the simulation, is it because we still haven't defined it?